sci-electronics/xcircuit/xcircuit-3.9.72.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Circuit drawing and schematic capture program"
+SRC_URI="http://opencircuitdesign.com/xcircuit/archive/${P}.tgz"
+HOMEPAGE="http://opencircuitdesign.com/xcircuit"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="tcl"
+
+DEPEND="
+	app-text/ghostscript-gpl:0=
```

Comment:
This and each line below: My last review gave you precise slot op information. Not for all DEPENDs.